expat/configure.ac

```diff
@@ -276,12 +276,46 @@ AS_IF([test "x${DOCBOOK_TO_MAN}" != x -a "x$with_docbook" != xno],
 
 AM_CONDITIONAL(WITH_DOCBOOK, [test "x${DOCBOOK_TO_MAN}" != x])
 
+AC_ARG_ENABLE(stdcxx,
+AS_HELP_STRING([--enable-stdcxx], [Specify CXX standard library]),
+    [AX_APPEND_COMPILE_FLAGS([-std=${enableval}], [AM_CXXFLAGS])],[])
+
+AC_ARG_ENABLE(fuzztargets,
+    AS_HELP_STRING([--enable-fuzztargets], [Enable fuzz targets]),[enable_fuzztargets=$enableval],[enable_fuzztargets=no])
+AM_CONDITIONAL([BUILD_FUZZTARGETS], [test "x$enable_fuzztargets" = "xyes"])
+AS_IF([test "x$enable_fuzztargets" = "xyes"], [
+    AC_MSG_CHECKING([whether $CXX accepts -fsanitize=fuzzer])
+    AC_LANG_PUSH(C++)
+    AS_VAR_COPY([OLDFLAGS], [CXXFLAGS])
+    AX_APPEND_FLAG([-fsanitize=fuzzer], [CXXFLAGS])
+    AC_LINK_IFELSE([AC_LANG_SOURCE([[
+#include <sys/types.h>
+extern "C" int LLVMFuzzerTestOneInput(const unsigned char *Data, size_t Size);
+extern "C" int LLVMFuzzerTestOneInput(const unsigned char *Data, size_t Size) {
+(void)Data;
+(void)Size;
+return 0;
+}
+        ]])],
+        [ AC_MSG_RESULT(yes)
+          AX_APPEND_COMPILE_FLAGS([-fsanitize=fuzzer-no-link], [CFLAGS])
+          AX_APPEND_COMPILE_FLAGS([-fsanitize=fuzzer-no-link], [CXXFLAGS])
+          has_sanitizefuzzer=yes],
+        [ AC_MSG_RESULT(no)
+          AC_MSG_ERROR([Fuzz targets require a compiler accepting option -fsanitize=fuzzer])]
+    )
+    AS_VAR_COPY([CXXFLAGS], [OLDFLAGS])
+    AC_LANG_POP()
+])
+AM_CONDITIONAL([HAS_SANITIZEFUZZER], [test "x$has_sanitizefuzzer" = "xyes"])
```

Comment:
We have `configure.ac` formatting rules in #306 by now. Would be great to see them applied. Thanks!